go/integTest/glide_test_suite_test.go

```diff
@@ -279,6 +301,31 @@ func (suite *GlideTestSuite) clusterClient(config *api.GlideClusterClientConfigu
 	return client
 }
 
+func (suite *GlideTestSuite) createConnectionTimeoutClient(
+	clusterMode bool,
+	connectTimeout, requestTimeout int,
+	backoffStrategy *api.BackoffStrategy,
+) (api.BaseClient, error) {
+	if clusterMode {
+		clientConfig := api.NewGlideClusterClientConfiguration().
+			WithAddress(&suite.clusterHosts[0]).
+			WithUseTLS(suite.tls).
+			WithAdvancedConfiguration(
+				api.NewAdvancedGlideClusterClientConfiguration().WithConnectionTimeout(connectTimeout)).
+			WithRequestTimeout(requestTimeout)
```

Comment:
you can introduce `defaultClusterClientConfig()` and `defaultStandaloneClientConfig()` and ...